packages/mikado_reborn/src/components/Switch/Switch.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <div>
+    <span
+      class="mkr__switch"
+      :class="{ 'mkr__switch--active': value }"
+      role="checkbox"
+      :aria-checked="!!value ? true : false"
```

Comment:
If value is a boolean, we can safely use it here
```suggestion
      :aria-checked="value"
```